pages/book/operators.mdx

```diff
@@ -107,17 +164,36 @@ pow(2, 255) * pow(2, 255); // build error: integer overflow!
 
 #### Divide, `/` [#binary-divide]
 
-Binary slash (_division_) operator `/{:tact}` is used for integer division of two values, which truncates towards zero. An attempt to divide by zero would result in an error with [exit code 4](/book/exit-codes#4): `Integer overflow`.
+Binary slash (_division_) operator `/{:tact}` is used for integer division of two values, which truncates towards zero if result is positive, and away from zero if result is negative. This is also called [rounding down](https://en.wikipedia.org/wiki/Rounding#Rounding_down) (or rounding towards $-∞$), where, for example, result value of $23.7$ gets rounded to $23$, and $−23.2$ gets rounded to $−24$.
+
+An attempt to divide by zero would result in an error with [exit code 4](/book/exit-codes#4): `Integer overflow`.
 
 Can only be applied to values of type [`Int{:tact}`][int]:
 
 ```tact
 let two: Int = 2;
 two / 2; // 1
-two / 1; // 2
+two / 1; // 0
 -1 / 5;  // -1
+-1 / -5; // 0
+1 / -5;  // -1
+1 / 5;   // 0
+6 / 5;   // 1, rounding down
+-6 / 5;  // -2, rounding down (towards -∞)
 ```
 
+<Callout>
+
+  Because division operator `/{:tact}` in Tact rounds down, the following expression always holds `true{:tact}`:
```

Comment:
```suggestion
  Note the following relationship between the division and modulo operators always holds:
```

Reply:
If division operator wouldn't round down and produced real numbers, that wouldn't work. Euclidean-ish thingy you're suggesting only works with Ints. I may rewrite that part a bit different, mentioning that Tact & TON only operate on integer numbers